environment/terraform.tfvars.json

```diff
@@ -35,7 +35,7 @@
       "symfony_env": "prod",
       "db_subnet_group": "private",
       "ec_subnet_group": "private",
-      "sirius_api_account": "649098267436",
+      "sirius_api_account": "492687888235",
```

Comment:
Should production not still be pointing at production?

Reply:
It is. I think it might be the way it's collapsed. It's the master and preproduction ones i changed

Reply:
Misread the diff, apologies 🤦